tests/datasource/fluent/test_pandas_datasource.py

```diff
@@ -474,17 +475,26 @@ def test_read_dataframe(empty_data_context: AbstractDataContext, test_df_pandas:
 @pytest.mark.cloud
 def test_cloud_get_csv_asset_not_in_memory(valid_file_path: pathlib.Path):
     # this test runs end-to-end in a real Cloud Data Context
-    context = gx.get_context(mode="cloud")
-    csv_asset_name = f"DA_{uuid.uuid4().hex}"
-    datasource = context.data_sources.pandas_default
-    _ = datasource.add_csv_asset(
-        name=csv_asset_name,
-        filepath_or_buffer=valid_file_path,
+    context = gx.get_context(
+        mode="cloud",
+        cloud_base_url=os.environ.get("GX_CLOUD_BASE_URL"),
+        cloud_organization_id=os.environ.get("GX_CLOUD_ORGANIZATION_ID"),
+        cloud_access_token=os.environ.get("GX_CLOUD_ACCESS_TOKEN"),
     )
-    csv_asset = datasource.get_asset(name=csv_asset_name)
-    csv_asset.build_batch_request()
+    datasource_name = f"DS_{uuid.uuid4().hex}"
+    csv_asset_name = f"DA_{uuid.uuid4().hex}"
+    try:
+        datasource = context.data_sources.add_pandas(name=datasource_name)
```

Comment:
nit: The add_pandas line should be pulled out of the try block because you can't delete it if it doesn't get successfully created.

Reply:
Right